src/test/cli.build.test.ts

```diff
@@ -170,5 +171,33 @@ describe('Dev Containers CLI', function () {
 			assert.equal(response.imageName[0], image1);
 			assert.equal(response.imageName[1], image2);
 		});
+
+		it('should fail with --push true and --output', async () => {
+			let success = false;
+			const testFolder = `${__dirname}/configs/dockerfile-with-target`;
+			try {
+				await shellExec(`${cli} build --workspace-folder ${testFolder} --output type=oci,dest=output.tar --push true`);
+			} catch (error) {
+				assert.equal(error.error.code, 1, 'Should fail with exit code 1');
+				const res = JSON.parse(error.stdout);
+				assert.equal(res.outcome, 'error');
+				assert.match(res.message, /cannot be used with/);
+			}
+			assert.equal(success, false, 'expect non-successful call');
+		});
+
+		it('file /tmp/output.tar should exist when using --output type=oci,dest=/tmp/output.tar', async () => {
+			const testFolder = `${__dirname}/configs/dockerfile-with-target`; 
+			const outputPath = `/tmp/output.tar`;
+			await shellExec(`docker buildx create --name ocitest'`);
```

Comment:
Seeing an extra single quote at the end of the string.

Could you replace the use of `/tmp` with `os.tmpdir()`? That will make it work on macOS too. Thanks!

Reply:
Sorry about the messed up quotes, and good call on using os.tmpdir(). Both should be fixed.